cg/services/order_validation_service/errors/case_sample_errors.py

```diff
@@ -158,3 +158,8 @@ class ContainerNameRepeatedError(CaseSampleError):
 class CaseNameSampleNameSameError(CaseSampleError):
     field: str = "name"
     message: str = "Sample name must be different from case name"
+
+
+class AllSamplesUnknownStatusError(CaseSampleError):
```

Comment:
```suggestion
class StatusUnknownError(CaseSampleError):
```
A bit clunkier but I like having the errors start with the "field" that they refer to, in this case "status".  Feel free to disagree though (there are probably some which break this rule already)